spec/models/custom_button_spec.rb

```diff
@@ -325,6 +327,30 @@
           )
         end
       end
+
+      %i(invoke invoke_async).each do |method|
+        describe "##{method}" do
+          it "publishes CustomButtonEvents with multiple vms" do
+            Timecop.freeze(Time.now.utc) do
```

Comment:
How is time related to this?

Reply:
Because [custom button events have timestamps](https://github.com/ManageIQ/manageiq/commit/54c0388eda97c6891525e81796984cb706008c71).